package.json

```diff
@@ -13,7 +13,7 @@
     "mock": "MOCK=true yarn start",
     "build": "rimraf ./dist && NODE_ENV=production webpack --mode production",
     "lint": "eslint src --ext .ts,.tsx,.js",
-    "lint:fix": "eslint src --fix . --ext .ts,.tsx,.js",
```

Comment:
I overlooked that we don't lint with that the tests. Is this intended?

Reply:
Don't know if intended, don't remember, the problem was the dot after `--fix`

Reply:
Yep, I understand. Just asking, cause this would allow to commit code with formatting issues in the tests